### PR TITLE
[feat] workflow-pr-review: narrative summary in Step 7 review body

### LINKED ISSUE #213: [feat] workflow-pr-review: include inline comment digest in Step 7 review body
## Goal

The Step 7 review body should contain the reviewer agent's narrative summary — not just a bare Posted N / deduped M count — so the top-level comment reads like a human peer-review and inline threads provide the detail.

## Motivation

Currently Step 7 submits:

> Reviewed by \`reviewer\` ([swe-workbench](https://github.com/lugassawan/swe-workbench)). Posted N inline comments, deduped M.

The reviewer agent already produces a prose narrative at the top of its output (overall assessment, key positives, critical findings). That narrative is discarded; only the count reaches the PR. Reviewers and authors must open every inline thread to understand the review.

## Surface

- [x] Skill — `workflow-*`

## Sketch

Expected GitHub PR review output (top-level comment + separate inline threads):

```
## Review Summary

Overall the implementation is solid — caching layer and fallback logic are well-structured,
and unit test coverage is thorough. One unhandled exception on the cache-hit path can cause
silent order-creation failures; a test-coverage gap exists for the boundary condition in
`shouldDeferOrder`. Detailed feedback in inline comments.

**Review Decision: COMMENT**
```

*Then, separately in the diff, inline comment threads appear attached to the relevant code lines — exactly as today.*

The orchestrator extracts the prose paragraphs above the `Severity | File:Line | ...` table from the reviewer agent's output, and uses that as the `--body` of `gh pr review`.

## Acceptance

- [ ] Step 7 `gh pr review --body` contains the reviewer's narrative summary (the prose above the findings table)
- [ ] Body ends with "Detailed feedback in inline comments." and `**Review Decision: COMMENT|APPROVE**`
- [ ] The "Reviewed by" byline (with hyperlinked `swe-workbench`) is kept, e.g. as a footer or sub-heading
- [ ] Graceful fallback when no narrative prose is present: falls back to current "Posted N / deduped M" format

### PR DESCRIPTION
**Pull request by lugassawan**
## Summary
- `skills/workflow-pr-review/SKILL.md` — Step 4: adds a third load-bearing "Narrative instruction" asking the reviewer to begin with a `## Review Summary` block (2–4 sentences: overall posture, strongest positives, top concerns). Adds a `$REVIEWER_OUTPUT` capture note.
- `skills/workflow-pr-review/SKILL.md` — Step 7: replaces the static one-liner body with an awk-extraction block that extracts prose above the first findings row or severity heading, builds a rich `## Review Summary` review body, and gracefully falls back to the legacy one-liner when no narrative prose is present.
- `agents/reviewer.md` — adds `## Review Summary (when instructed)` section mirroring the existing `## Decision footer (when instructed)` opt-in pattern.

## Test Plan

- [x] Changed skills/commands/agents load without errors
- [x] Examples in the diff were exercised manually
- [x] `bash scripts/validate.sh` — exit 0, "All checks passed"
- [x] `python3 -m pytest tests/ -q` — 357/357 pass

Closes #213

### CODE REVIEW

File: skills/workflow-pr-review/SKILL.md [hunk 1]
```diff
@@ -81,8 +81,12 @@ Pass the agent:
   > "Emit **repo-relative** paths in every finding (e.g. `src/foo.ts:42`, NOT `$WT/src/foo.ts:42`). The orchestrator uses these paths to position GitHub comments."
 - Footer instruction (load-bearing — opt-in per the agent's `## Decision footer (when instructed)` block):
   > "End the review with EXACTLY ONE of `**Review Decision: APPROVE**` or `**Review Decision: COMMENT**` on its own line, no prefix or trailing text. Never `REQUEST_CHANGES`."
+- Narrative instruction (load-bearing — opt-in per the agent's `## Review Summary (when instructed)` block):
+  > "Begin the review with a `## Review Summary` section: 2–4 sentences capturing overall posture, the strongest positives, and the most important concerns. The orchestrator uses these paragraphs as the top-level PR review body. Do not repeat per-finding detail there — that goes in the severity-grouped findings below."
 - Ticket-context prelude (if Step 3 produced one).
 
+Store the agent's complete text response as a shell variable: `REVIEWER_OUTPUT=<agent response>`. Step 7's awk block reads from this variable via `<<< "$REVIEWER_OUTPUT"`.
```
Comment (lugassawan) [inline skills/workflow-pr-review/SKILL.md]: **Low** — `REVIEWER_OUTPUT=<agent response>` is pseudo-code: it conveys intent but gives no concrete mechanism. In an LLM-orchestrator flow the assignment is runtime-implicit; a brief clarifying note such as _"In practice the orchestrator assigns the subagent's full text reply to this variable before the awk block runs"_ would make the capture contract explicit for future maintainers.

**Review by lugassawan — COMMENTED** [review]
Reviewed by `reviewer` (swe-workbench). Posted 1 inline comment, deduped 0.
